src/styles/blocks/main.scss

```diff
@@ -0,0 +1,21 @@
+.main {
+  margin: 0;
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: 200px;
+  justify-content: center;
```

Comment:
You have centered the grid items with `justify-content: center`, which is good for alignment. However, ensure that this aligns with the design requirements, as sometimes items may need to start from the left or have a different alignment.